app/views/carts/index.html.erb

```diff
@@ -8,8 +8,9 @@
     <p>Price: $<%= item.price %></p>
     <p>Quantity: <%= quantity %></p>
     <p>Subtotal: $<%= quantity * item.price %></p>
-    <p><%= link_to "Decrease quantity", cart_path(item), action: :decrease, method: :put %>
-    <p><%= link_to "Remove", cart_path(item), method: :delete %></p>
+    <p><%= link_to "Decrease quantity", cart_path(item), action: :decrease, method: :put %></p>
+    <p><%= link_to "Increase quantity", cart_path(item), action: :increase, method: :patch %></p>
```

Comment:
@rickyamparo @JunePaloma let's start thinking about how we want to render the link/button in the front-end. It seems too long to display "decrease quantity" or "increase quantity" in the cart. Let's change it to "up" or "down", "increase" or "decrease", or the like.